ansible/doc/binding-strategy.rst

```diff
@@ -0,0 +1,33 @@
+======================
+Binding Strategy Guide
+======================
+
+Overview
+========
+
+In OVN, a logical port can be bind to a local OVS port on any chassis/hypervisor, depending on the VM scheduler (e.g., ``nova-scheduler``). The binding strategy potentially impacts the network performance. That is binding all logical ports in a logical network on a single hypervisor performs differently than distributing the ports on multiple hypervisors.
```

Comment:
It would be good to linewrap this file at 80 characters. Can you do that please?